src/main/kotlin/bd/clinic/modules/order/OrderService.kt

```diff
@@ -30,11 +41,39 @@ class OrderService(
     fun addPatientOrder(order: OrderDTO): OrderDTO {
         checkIfOrderNumberIsUnique(order.orderNumber)
         val patient = patientService.findPatientOrThrow(order.patientId)
+        sendPostRequestToAllLaboratories(order)
+
         return OrderDTO(orderRepository.save(order.toOrderEntity(patient)))
     }
 
+    private fun sendPostRequestToAllLaboratories(orderDTO: OrderDTO) {
+        LabServiceClient.infrastructurePortMap.forEach { (laboratoryId, port) ->
+            val infrastructureOrder = orderDTO.copy(
+                    examinations = orderDTO.examinations.filter { it.laboratoryId == laboratoryId })
+            if (infrastructureOrder.examinations.isNotEmpty())
+                labServiceClient.sendRequest(infrastructureOrder, port)
+        }
+    }
+
+    private fun sendGetRequestToAllLaboratories(orderDTO: OrderDTO): OrderDTO {
+        val examinationResultList = mutableListOf<ExaminationDTO>()
```

Comment:
tu nie zwracamy `examinationResult` tylko to `examinationInfo`, z tego co jest tu na razie. nie wiem czy tak chcemy

Reply:
plus nie musisz dawac List na koncu nazwy, moze byc po prostu laboratories np., ale to juz sie czepiam :P wiadomo ze to lista

Reply:
a nie List<ExaminationResult>?

Reply:
> plus nie musisz dawac List na koncu nazwy, moze byc po prostu laboratories np., ale to juz sie czepiam :P wiadomo ze to lista

sdihvfghnibgjhnvjnfcndsanvdgnnjnjcaf

Reply:
ale fajny czat sobie znaleźliście